src/components/About.tsx

```diff
@@ -0,0 +1,44 @@
+type AboutProps = {
+  photoUrl?: string // optional for now
```

Comment:
This doesn't need to be optional if you set the photoUrl to the placehold.co image in the parent component.